src/desktop/apps/artwork/components/commercial/view.coffee

```diff
@@ -96,6 +97,30 @@ module.exports = class ArtworkCommercialView extends Backbone.View
         mode: 'login'
         trigger: 'click'
         redirectTo: location.href
+  
+  offer: (e) ->
+    e.preventDefault()
+
+    loggedInUser = CurrentUser.orNull()
+
+    serializer = new Serializer @$('form')
+    data = serializer.data()
+    editionSetId = data.edition_set_id
+    $target = $(e.currentTarget)
+
+    # If this artwork has an edition set of 1, send that in the mutation as well
+    if @artwork.get('edition_sets')?.length && @artwork.get('edition_sets').length == 1
+      editionSetId = @artwork.get('edition_sets')[0] && @artwork.get('edition_sets')[0].id
+
+    if loggedInUser
+      console.log("making offer!")
+    else
+      return mediator.trigger 'open:auth',
+        intent: 'buy now'
```

Comment:
Should any of these intent fields be "make offer"?

Reply:
🙌 yes good catch!